src/pages/informationmodel-details-page/infomodel-structure/styled.ts

```diff
@@ -17,16 +17,19 @@ const Title = styled.h2`
   overflow: hidden;
   text-overflow: ellipsis;
   margin-bottom: 1em;
+  font-size: 24px;
+  font-weight: 600;
 `;
 
 const Section = styled.div`
   margin-bottom: 1em;
 `;
 
 const SectionHeader = styled.h3`
-  color: ${({ theme }) => theme.colors.neutralDark};
+  color: ${({ theme }) => theme.extendedColors.neutralDark};
+  font-size: 20px;
   font-weight: 300;
-  margin-bottom: 1em;
+  margin-bottom: 20px;
```

Comment:
En fordel å bruke 'em' for margins og paddings da det vil følge relativt til font-size. Om man f.eks. endrer font-size på komponent i en media-query så behøver man ikke manuelt sette margin da den vil bli i forhold til ny font-size => mindre styling å vedlikeholde.

Reply:
Fikset